src/core/bitop.d

```diff
@@ -548,37 +667,11 @@ void volatileStore(ulong * ptr, ulong  value);   /// ditto
             ret;
         }
     }
-    else
-    {
-        // swap odd and even bits
-        x = ((x >> 1) & 0x5555_5555) | ((x & 0x5555_5555) << 1);
-        // swap consecutive pairs
-        x = ((x >> 2) & 0x3333_3333) | ((x & 0x3333_3333) << 2);
-        // swap nibbles
-        x = ((x >> 4) & 0x0F0F_0F0F) | ((x & 0x0F0F_0F0F) << 4);
-        // swap bytes
-        x = ((x >> 8) & 0x00FF_00FF) | ((x & 0x00FF_00FF) << 8);
-        // swap 2-byte long pairs
-        x = ( x >> 16              ) | ( x               << 16);
-        return x;
-
-    }
```

Comment:
@MartinNowak I'm not an assembly expert, so someone else should file a separate pull request if the `asmBitswap` functions need an optimization pass.

I'm mostly just trying to standardize the `core.bitop` API so that all functions are available, regardless of the execution environment.